### PR TITLE
PRM-2984 New Overpayment Charge added + content for summary card

### PR DESCRIPTION
**Pull request by AndrewRButcher**

### CODE REVIEW

File: app/viewmodels/SummaryCardHelper.scala [hunk 1]
```diff
@@ -407,6 +407,7 @@ class SummaryCardHelper @Inject()() extends ImplicitDateFormatter with ViewUtils
       case ProtectiveAssessmentFirstLPP | ProtectiveAssessmentSecondLPP | ProtectiveAssessment => "summaryCard.lpp.key3.value.protectiveAssessment"
       case POAReturnChargeFirstLPP | POAReturnChargeSecondLPP | POAReturnCharge => "summaryCard.lpp.key3.value.poaReturnCharge"
       case AAReturnChargeFirstLPP | AAReturnChargeSecondLPP | AAReturnCharge => "summaryCard.lpp.key3.value.aaReturnCharge"
+      case VATOverpaymentForTax => ""
```
Comment (LewisMitchell) [inline app/viewmodels/SummaryCardHelper.scala:410]: Why is this blank?

File: app/models/lpp/MainTransactionEnum.scala [hunk 2]
```diff
@@ -37,6 +37,7 @@ object MainTransactionEnum extends Enumeration {
   val ProtectiveAssessment: MainTransactionEnum.Value = Value("4733")
   val ProtectiveAssessmentFirstLPP: MainTransactionEnum.Value = Value("4761")
   val ProtectiveAssessmentSecondLPP: MainTransactionEnum.Value = Value("4762")
+  val VATOverpaymentForTax: MainTransactionEnum.Value = Value("4764")
```
Comment (LewisMitchell) [inline app/models/lpp/MainTransactionEnum.scala:40]: Could there be a test for this?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

**Comment by AndrewRButcher** [comment]
addressed PR comments

**Comment by LewisMitchell** [comment]
image

**Comment by LewisMitchell** [comment]
Welsh being added as part of a different task

**Pull request merged**